cli/flags.rs

```diff
@@ -1082,6 +1102,72 @@ Future runs of this module will trigger no downloads or compilation unless
     )
 }
 
+fn coverage_subcommand<'a, 'b>() -> App<'a, 'b> {
+  SubCommand::with_name("coverage")
+    .about("Print coverage reports")
+    .long_about(
+      "Print coverage reports from coverage profiles.
+
```

Comment:
Let's make this subcommand unstable for now

Reply:
I see that it's checked in `main.rs`, can we add `require("unstable")` here instead?